### ISSUE TITLE
schematics for 3V3 MCU:

### ISSUE THREAD

**Issue by mullets** [tomaskovacik/VAGFISReader#1]
man, this schematics is a little confused.  you do not have another vesrion? maybe in jpg? or pics?

**Comment by tomaskovacik** [comment]
what exactly? I will do not draw schematics for few simple voltage dividers ...

**Comment by mullets** [comment]
Could you help me please? I'm in a instalation and I do not know where I can connect de wires by 5v(or 3v, I don't understand your explanation:

"FULLY TESTED: full fis mode on atmega328,radio mode on stm32 (bluepill board) over 3v resistor devider (10k to 47k) with 10k pullup on data and clk line, these pull up are required! I checked lot of schematics of radios, and most of them do not have internall pullups, so if it works without nice ,but mostly it will not. In case of 5V board (AVR MCU) internal pull up is ok, in case of 3v3 mcu external pull up + divider is erquired, if transistor based level shifter with internal pullup is used, check what value are these pullups and make stronger pulldown (half value resistor will do) on both low a high level side of ENABLE line (tested with mosfet based level shifter)"

My doubts: I'm using a bose scheme. Which resistor and voltage I will use? I understood you use 3 cases with differences of voltage, resistors and transistors.
I've tried to connect with 5v but I don't know where I will use the 5v wire.

Please, help me :/

**Comment by tomaskovacik** [comment]
Hello,What mcu are you using? What exactly did you try and did not work?On ascii representation of "how to connect it" are pins marked as radio which goes to radio, then there is side marked mcu which goes to mcu( but only 3v3 mcu)And then there are resistors connected to lines which goes from radio  and other side of resistor goes to 5v power suply marked as 5VWhat is not clear on this?ThxT.On 9 Feb 2019 15:39, Rafael Moreira D Andrea <notifications@github.com> wrote:Could you help me please? I'm in a instalation and I do not know where I can connect de wires by 5v(or 3v, I don't understand your explanation:
"FULLY TESTED: full fis mode on atmega328,radio mode on stm32 (bluepill board) over 3v resistor devider (10k to 47k) with 10k pullup on data and clk line, these pull up are required! I checked lot of schematics of radios, and most of them do not have internall pullups, so if it works without nice ,but mostly it will not. In case of 5V board (AVR MCU) internal pull up is ok, in case of 3v3 mcu external pull up + divider is erquired, if transistor based level shifter with internal pullup is used, check what value are these pullups and make stronger pulldown (half value resistor will do) on both low a high level side of ENABLE line (tested with mosfet based level shifter)"
My doubts: I'm using a bose scheme. Which resistor and voltage I will use? I understood you use 3 cases with differences of voltage, resistors and transistors.
I've tried to connect with 5v but I don't know where I will use the 5v wire.
Please, help me :/

**Comment by tomaskovacik** [comment]
https://github.com/tomaskovacik/VAGFISReader/wiki/How-to-connect

check this link
